### PR TITLE
RAWACF updates

### PR DESCRIPTION
**Pull request by mts299**
This PR focuses on fixing some bugs and updating SuperDARN Radars for getting new JME and DCN hardware files. 

**NOTE**: I am looking into a new method for getting radar info via radar.dat file so these things do not happen in the future. 

The RAWACF fixes where adding optional fields to read in various radar RAWACF files. 
One test that can be done is literally reading in DCN file as it some times is missing `acfd` field and that is apparently okay!

### CODE REVIEW

**Comment by mts299** [comment]
@ksterne looks like I am committing some of the hardware files I had to add into the subrepo as it was outdated would it be possible for you to make sure I am not screwing anything up with that commit?

Also I will be looking at why pyDARN didn't pull from the most recent master.

**Comment by mts299** [comment]
Please review PR #46 first as this fixes the sub-repo problem.

**Comment by mts299** [comment]
Reverted the hardware commits and merged `develop` back in with the setup.py bug fix. Does anyone want to test this? @Cooper10 @kevinkrieger @carleyjmartin

File: pydarn/io/superdarn_formats.py [hunk 1]
```diff
@@ -66,11 +66,16 @@ class Rawacf():
...
-        'pwr0': 'f',
-        'acfd': 'f',
+        'pwr0': 'f'
         }
-    extra_fields = {
-            'ifmode': 'h',
+    correlation_field = {
+        'acfd': 'f'
+        }
+
```
Comment (kevinkrieger) [inline pydarn/io/superdarn_formats.py:74]:
```suggestion
```

File: pydarn/utils/superdarn_radars.py [hunk 2]
```diff
@@ -343,6 +346,9 @@ class SuperDARNRadars():
               96: _Radar('Dome C',
```
Comment (kevinkrieger) [inline pydarn/utils/superdarn_radars.py:346]:
```suggestion
              96: _Radar('Dome C East',
```

File: pydarn/io/superdarn_formats.py [hunk 3]
```diff
@@ -66,11 +66,16 @@ class Rawacf():
...
+    digitizing_field = {
+            'ifmode': 'h'
+            }
+    fittex_field = {
             'mplgexs': 'h',
```
Comment (kevinkrieger) [inline pydarn/io/superdarn_formats.py:79]:
```suggestion
            'mplgexs': 'h'
```

File: pydarn/io/superdarn.py [hunk 4]
```diff
@@ -414,8 +414,10 @@ def read_rawacf(self) -> List[dict]:
         pydarn_log.debug("Reading Rawacf file: {}".format(self.dmap_file))
 
         file_struct_list = [superdarn_formats.Rawacf.types,
-                            superdarn_formats.Rawacf.extra_fields,
-                            superdarn_formats.Rawacf.cross_correlation_field]
+                            superdarn_formats.Rawacf.correlation_field,
+                            superdarn_formats.Rawacf.cross_correlation_field,
```
Comment (kevinkrieger) [inline pydarn/io/superdarn.py:418]: I just need to find some files to test these changes with

**Comment by kevinkrieger** [comment]
I've tested this PR with all rawacf files available from January 1st 2020, it successfully read each file without throwing any exceptions. The test code used is:

`
import pydarn
import sys
from glob import glob

file_glob = glob('*rawacf')

for f in file_glob:
    print(f)
    sd = pydarn.SDarnRead(f)
    try:
        r = sd.read_rawacf()
    except Error as e:
        print(e)
`

**Comment by mts299** [comment]
@kevinkrieger would you like to merge this code :)

**Comment by mts299** [comment]
@kevinkrieger I got Travis CI to be happy again!

**Pull request merged**